src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/Header.js

```diff
@@ -96,7 +96,7 @@ export default class Header extends React.PureComponent<Props> {
                     <Checkbox
                         checked={allSelected}
                         onChange={this.handleAllSelectionChange}
-                        skin={skin}
+                        skin={skin === 'dark' ? 'light' : 'dark'}
```

Comment:
Why is this line necessary? As far as I see this will not change anything :thinking:

Reply:
it sets the skin of the checkbox to `light` if the skin of the header is `dark` and vice versa. this only affects tables that render the checkbox into the first cell (eg. the category list).